lesson_08/homework_08.py

```diff
@@ -0,0 +1,15 @@
+numbers_list: list[str] = ["1,2,3,4", "1,2,3,4,50", "qwerty1,2,3"]
+print(numbers_list)
+
+def sum_number_list(list_of_numbers: list[str]) -> None:
+    result: list[int] = list()
+    try:
+        for item in list_of_numbers:
+            chars_list: list = item.split(",")
+            result.append(sum(int(integer) for integer in chars_list))
```

Comment:
Again. Please - use List comprehension for list assigning instead of for loop with append func.